app/scripts/migrations/105.ts

```diff
@@ -106,26 +127,33 @@ function createSelectedAccountForAccountsController(
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   state: Record<string, any>,
 ) {
-  const selectedAddress = state.PreferencesController?.selectedAddress;
+  const selectedAddress = undefined; // state.PreferencesController?.selectedAddress;
```

Comment:
Shouldn't we keep the `PreferencesController.selectedAddress` here? If users do not face the issue (of having no selected address), then the migration would go through the "naturally".

We only want to cover users that fall into the "undefined" case, no?